README.markdown

```diff
@@ -6543,6 +6547,82 @@ See also [ngx.shared.DICT](#ngxshareddict).
 
 [Back to TOC](#nginx-api-for-lua)
 
+ngx.shared.DICT.ttl
+-------------------
+**syntax:** *ttl, err = ngx.shared.DICT:ttl(key)*
+
+**context:** *init_by_lua&#42;, set_by_lua&#42;, rewrite_by_lua&#42;, access_by_lua&#42;, content_by_lua&#42;, header_filter_by_lua&#42;, body_filter_by_lua&#42;, log_by_lua&#42;, ngx.timer.&#42;, balancer_by_lua&#42;, ssl_certificate_by_lua&#42;, ssl_session_fetch_by_lua&#42;, ssl_session_store_by_lua&#42;*
+
+**requires:** `resty.core.shdict` or `resty.core`
```

Comment:
My initial idea is to introduce this new `requires` section to make this obvious early in the reading of this method's documentation. There is also a note at the bottom of it with a link to the lua-resty-core library. Open to suggestions!

Reply:
@thibaultcha That's fine, though we'll soon make lua-resty-core mandatory and loaded by default in the next `ngx_lua` release anyway :)

Reply:
@agentzh Yeah - I remember :) Alright then!